subprojects/testfx-core/src/main/java/org/testfx/robot/impl/BaseRobotImpl.java

```diff
@@ -42,6 +42,13 @@
     private static final String PROPERTY_TESTFX_ROBOT_AWT = "awt";
     private static final String PROPERTY_TESTFX_ROBOT_GLASS = "glass";
 
+    private static final KeyCode OS_SPECIFIC_SHORTCUT;
+
+    static {
+        String osName = System.getProperty("os.name");
```

Comment:
Can you make this: `String osName = System.getProperty("os.name").toLowerCase(Locale.US)` and then below use

`osName.startsWith("mac")`